scripts/classify_resources_by_owners.py

```diff
@@ -101,22 +118,22 @@ def _process_users(self):
             roles = self.map.roles_by_user[user.id]
             user_type = None
             for rol in roles:
-                if rol[0] == BASIC_ROLE_ID:
+                if rol == BASIC_ROLE_ID:
                     user_type = 'basic'
```

Comment:
Probably in the past is was used for other purposes, now it may be replaced with a boolean to check is the user has type. Changed to user_has_type boolean.